source/common/http/conn_manager_impl.cc

```diff
@@ -369,6 +369,8 @@ ConnectionManagerImpl::ActiveStream::ActiveStream(ConnectionManagerImpl& connect
   }
   stream_info_.setDownstreamLocalAddress(
       connection_manager_.read_callbacks_->connection().localAddress());
+  stream_info_.setDownstreamDirectlyConnectedAddress(
```

Comment:
I think I would prefer `downstreamDirectRemoteAddress()` for consistency with existing naming, or something like that.